src/zeam/form/base/sorters.py

```diff
@@ -1,35 +1,56 @@
-# -*- coding: utf-8 -*-
+def sort_components(order):
 
-import operator
-
-def sort_components(components, order, reverse=False):
     if not isinstance(order, list) and not isinstance(order, tuple):
         raise ValueError(
             "Please provide a valid list or tuple of component identifiers.")
+
     if len(order) < 2:
         raise ValueError(
             "Please provide a list of, at least, two component identifiers.")
 
-    def compare(c1, c2, order=order):
+    order = tuple(order)
 
+    def compare(c1, c2):
```

Comment:
All of this no longer makes sense. There's no compare method in Python3. It is better to replace this by something that generates a function that you can use as `key` to sort?

Reply:
I thought this was what we were already doing, as this sort_components is used as key
> from zeam.form.base import sort_components
> s5.sort(key=sort_components(['uranus', 'venus', 'jupiter']))